src/integrator.jl

```diff
@@ -119,6 +119,6 @@ Tempering step. `step` is a named tuple with
 - `is_half` indicating whether or not it's (the first) half momentum/tempering step
 """
 function temper(lf::TemperedLeapfrog, r, step::NamedTuple{(:i, :is_half),<:Tuple{Integer,Bool}}, n_steps::Int)
-    i_temper = 2(step.i - 1) + 1 + step.is_half    # counter for half temper steps
+    i_temper = 2(step.i - 1) + 1 + !step.is_half    # counter for half temper steps
```

Comment:
Quite dumb bug. I was thinking maybe we should use the other appraoch in which we generate the tempering schedule before all leapfrog steps and simply apply them. With this appraoch we can simply "print out" the tempering schedule. And this appraoch is more general as the tempering schedule could be more flexible.

Reply:
we can probably do a dry run and print out the tempering variables without changing the code.

Reply:
We could. But how about using a speficic tempering schedule. E.g. allow user to provides any sequnece of alphas, e.g. TemperedLeapfrog(ϵ, [1.2,1.1,1.05,1.025])?

Reply:
Yeah, but maybe keep the current code for now, and change it when we find a good motivating use case.